### PR TITLE
major go-sdk update

### PR DESCRIPTION
**Pull request by svenwltr**

### CODE REVIEW

**Review by der-eismann — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-MIT License
+The MIT License (MIT)
 
 Copyright (c) 2019 reBuy reCommerce GmbH
```
Comment (bjoernhaeuser) [inline LICENSE:3]:
```suggestion
Copyright (c) 2020 reBuy reCommerce GmbH
```

File: .gitignore [hunk 2]
```diff
@@ -1 +1,6 @@
-/templates/generated
+/vendor
+/dist
+node-drainer*
+
+.terraform*
```
Comment (der-eismann) [inline .gitignore:5]: :lolwut:

File: cmd/templates/status.html [hunk 3]
```diff
@@ -0,0 +1,354 @@
+<!doctype html>
```
Comment (bjoernhaeuser) [inline cmd/templates/status.html:1]: Did you add this file on purpose?

File: Dockerfile [hunk 4]
```diff
@@ -1,8 +1,5 @@
...
-RUN apk add --no-cache diffutils git
+FROM quay.io/rebuy/rebuy-go-sdk:v2.4.0 as builder
 
-ADD . /golang-template
-WORKDIR /golang-template
-
-ENTRYPOINT ["/golang-template/entrypoint.sh"]
+FROM alpine:latest
+RUN apk add --no-cache ca-certificates
+COPY --from=builder /build/dist/node-drainer /usr/local/bin/
```
Comment (der-eismann) [inline Dockerfile:5]:
```suggestion
COPY --from=builder /build/dist/go-template /usr/local/bin/
```

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,3 @@
+PACKAGE=github.com/rebuy-de/node-drainer
```
Comment (der-eismann) [inline Makefile:1]:
```suggestion
PACKAGE=github.com/rebuy-de/go-template
```

**Comment by svenwltr** [comment]
I updated things.

**Review by bjoernhaeuser — APPROVED** [review]

**Pull request merged**